misc/debug-adapter/modules/debug-adapter-core/spotbugs-exclude.xml

```diff
@@ -20,11 +20,14 @@
     <Match>
         <Or>
             <And>
-                <Bug pattern="DLS_DEAD_LOCAL_STORE"/>
-                <And>
-                    <Class name="org.ballerinalang.debugadapter.JBallerinaDebugServer"/>
-                    <Method name="evaluate"/>
-                </And>
+                <Class name="org.ballerinalang.debugadapter.JBallerinaDebugServer"/>
+                <Or>
+                    <And>
+                        <Method name="evaluate"/>
+                        <Bug pattern="DLS_DEAD_LOCAL_STORE"/>
+                    </And>
+                    <Bug pattern="DM_EXIT"/>
```

Comment:
Can't we handle `DM_EXIT` error rather than adding to spotbugs-exclude.xml?

Reply:
Can you think of any other alternative instead of doing a `system.exit()` to terminate the debug server VM?

Reply:
We can use `system.exit()`. My suggestion is it's always better to handle spotbugs errors and try to avoid adding those into `spotbugs-exclude.xml`. +1 to add `DM_EXIT` if it is unavoidable.